build/db2-migration-test-docker.sh

```diff
@@ -7,13 +7,13 @@
 set -ex
 
 echo "Preparing environment for fhir-server integration tests..."
-if [[ -z "${WORKSPACE}" ]]; then
-    echo "ERROR: WORKSPACE environment variable not set!"
-    exit 2
-fi
+
+# The full path to the directory of this script, no matter where its called from
```

Comment:
I think what I built for persistence-automation for postgres, could play well here, and we should probably reconcile early as I think you have a `pattern` that I've just generalized.

Reply:
I find that having this run the same no matter where its invoked from simplifies the script, so I settled on this pattern based on https://stackoverflow.com/a/246128/161022

Previously, these scripts required us to set a WORKSPACE env variable, but this got me very confused when I cloned a second copy of our repo and tried running the scripts and couldn't understand why it wasn't using the right stuff.  Better to have it know its in a build dir under the project root and always execute relative to that.